doc/release-notes.md

```diff
@@ -66,6 +66,16 @@ New RPCs
 Build System
 ------------
 
+Files
+-----
+
+* On startup, the list of banned hosts and networks (via `setban` RPC) in
+  `banlist.dat` is ignored and only `banlist.json` is considered. Bitcoin Core
```

Comment:
suggestions:

- "(via the `setban` RPC or the GUI peers tab)"
-  "is now ignored and `banlist.json` is read instead.`
- s/double check/verify/

Reply:
Hm, the second sentence seems to contradict the first one. Maybe start with the explaining that if banlist.json is not found, it is created from the data in banlist.dat.

Reply:
might do if I have to retouch. Otherwise this can be edited in the wiki in about 6 months